auth-service/src/integrationTest/java/com/syed/authservice/GetUserIntegrationTest.java

```diff
@@ -0,0 +1,23 @@
+package com.syed.authservice;
+
+import com.syed.authservice.clients.IdentityServiceClient;
+import com.syed.authservice.domain.model.response.UserV2Response;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class GetUserIntegrationTest extends BaseTest {
+
+    @Autowired
+    private IdentityServiceClient identityServiceClient;
+
+    @Test
+    void getClient() {
```

Comment:
```suggestion
    void getUser() {
```

Reply:
resolved in [c4bd070](https://github.com/AbulSyed/locked-out/pull/52/commits/c4bd0704c63a68213bdbd5d701072540ec4a6b78)